config/database.yml

```diff
@@ -66,6 +66,8 @@ test:
   pool: 5
   username: wbooks-api
   password: wbooks-api
+  min_messages: WARNING
```

Comment:
Why? 🤔

Reply:
this was to prevent my macbook to thow many warnings, but didnt work

Reply:
Then.. remove it, please.